js/apps/admin-ui/src/realm-settings/localization/EffectiveMessageBundles.tsx

```diff
@@ -101,11 +107,20 @@ export const EffectiveMessageBundles = ({
         },
       );
 
-      return filter.hasWords.length > 0
-        ? messages.filter((m) =>
-            filter.hasWords.some((f) => m.value.includes(f)),
-          )
-        : messages;
+      const filteredMessages =
```

Comment:
It's not possible to do this kind of filtering on the API? @edewit

Reply:
thanks @jonkoops this implementation was suggested by @edewit

Reply:
Then I will assume it's not possible, thanks :+1: